msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md

```diff
@@ -23,45 +25,68 @@ Teams supports access to apps during meeting for the following meeting types:
+Meeting lifecycle includes pre-meeting, in-meeting, and post-meeting app experience, depending on the user type and user's role in Teams meeting.
 
-The following illustration gives you an idea of the meeting app extensibility features:
+## User types in Teams
 
-![Meeting app extensibility](../assets/images/apps-in-meetings/meetingappextensibility.png)
+User types, such as in-tenant, guest, federated or external user in a meeting can do one of the [User roles in Teams meeting](#user-roles-in-teams-meeting).
 
-This article provides an overview of meeting app extensibility, API references, enable and configure apps for meetings, and custom Together Mode scenes in Teams.
+The following list details the various user types along with their accessibility:
 
-> [!Note]
->
-> Meeting apps (side panel, meeting stage) are supported in Teams desktop client. Where as in Teams web client it is supported only when the developer preview is enabled.
+* **In-tenant**: In-tenant users belong to the organization and have credentials in Azure Active Directory (AAD) for the tenant. They're full-time, onsite, or remote employees. An in-tenant user can be an organizer, presenter, or attendee.
+* **Guest**: A guest is a participant from another organization invited to access Teams or other resources in the organization's tenant. Guests are added to the organization’s Azure AD and have same Teams capabilities as a native team member. They have access to team chats, meetings, and files. A guest can be an organizer, presenter, or attendee. For more information, see [guest access in Teams](/microsoftteams/guest-access).
+* **Federated or external**: A federated user is an external Teams user in another organization who has been invited to join a meeting. Federated users have valid credentials with federated partners and are authorized by Teams. They don't have access to your teams or other shared resources from your organization. Guest access is a better option for external users to have access to teams and channels. For more information, see [manage external access in Teams](/microsoftteams/manage-external-access).
+
+    > [!NOTE]
+    > Your Teams users can add apps when they host meetings or chats with other organizations. The users can use apps shared by external users when your users join meetings or chats hosted by other organizations. The data policies of the hosting user's organization, as well as the data sharing practices of the third-party apps shared by that user's organization, will be in effect.
+
+    > [!IMPORTANT]
+    > Currently, third-party apps are available in Government Community Cloud (GCC) but are not available for GCC-High and Department of Defense (DOD). Third-party apps are turned off by default for GCC. To turn on third-party apps for GCC, see [manage app permission policies](/microsoftteams/teams-app-permission-policies) and [manage apps](/microsoftteams/manage-apps).
```

Comment:
This sort of information (gov cloud) may be better in a Meeting FAQ page or closer to the bottom of the page, as it will only be relevant to a small minority of folks reading this page.

Reply:
@v-sdhakshina  pls mark this resolved